src/components/section-list.component.js

```diff
@@ -15,42 +16,49 @@ type Props = {
   buttonAction: Function,
 };
 
-const styles = StyleSheet.create({
-  section: {
-    marginTop: 15,
-  },
-  topHeader: {
-    flex: 1,
-    flexDirection: 'row',
-    justifyContent: 'space-between',
-    alignItems: 'center',
-  },
-  sectionTitle: {
-    color: colors.black,
-    padding: 15,
-    ...fonts.fontPrimaryBold,
+const Section = styled.View`
+  margin-top: 15;
+`;
+const Header = styled.View`
+  flex: auto;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+`;
+const SectionTitle = styled.Text`
+  color: ${colors.black};
+  padding: 15px;
+  ${fonts.fontPrimaryBold};
+`;
+const TitleView = styled.View`
+  padding: 15px;
+`;
+const StyledList = styled(List).attrs({
+  containerStyle: {
+    marginTop: 0,
+    borderBottomColor: colors.grey,
+    borderBottomWidth: 1,
   },
-  listTitle: {
+})``;
+const StyledListItem = styled(ListItem).attrs({
+  titleStyle: {
     color: colors.black,
     ...fonts.fontPrimary,
   },
-  button: {
+})``;
+const LoadingIcon = styled(ActivityIndicator)`
+  margin: 20px 0;
+`;
+const StyledButton = styled(Button).attrs({
+  buttonStyle: {
     backgroundColor: colors.white,
     borderColor: colors.primaryDark,
     borderWidth: 1,
-    borderRadius: 3,
     paddingVertical: 5,
     paddingHorizontal: 10,
+    ...fonts.fontPrimaryBold,
```

Comment:
`fonts.fontPrimaryBold` belongs to attr `textStyle`, and other attrs(fontSize/color) are missing.

Reply:
I tried to get fontSize into text style but it didn't seem to work.

Reply:
> Would colour go in buttonStyle or textStyle?

Neither. It belongs to an attr iteself, as well as fontSize.

```
const StyledButton = styled(Button).attrs({
  textStyle: {},
  fontSize: 13,
  color: colors.primaryDark,
  buttonStyle: {},
}``;

Reply:
Yeah awesome made the change!